test/common/quic/envoy_quic_proof_source_test.cc

```diff
@@ -1,5 +1,6 @@
 #include <memory>
 #include <string>
+#include <type_traits>
```

Comment:
I don't see anything from type_traits being introduced in this PR, can this be removed?

Reply:
removed